MixItUp.WPF/Controls/Chat/ChatUserControl.xaml.cs

```diff
@@ -14,31 +16,61 @@ public partial class ChatUserControl : UserControl
     {
         public UserViewModel User { get; private set; }
 
+        public ChatUserControl()
+        {
+            this.DataContextChanged += ChatUserControl_DataContextChanged;
+
+            InitializeComponent();
+        }
+
         public ChatUserControl(UserViewModel user)
```

Comment:
Not sure how you feel about "chaining" constructors together but you could do:
public Foo() {}
public Foo(Bar b) : this() {}

Reply:
Yea, makes more sense now. Didn't make as much with the previous path.